src/main.rs

```diff
@@ -1,10 +1,11 @@
-use std::{collections::HashMap, error::Error, fs, io::Write, num::ParseIntError, ops::AddAssign};
+use std::{collections::HashMap, error::Error, io::Write, num::ParseIntError, ops::AddAssign};
 
 fn main() -> Result<(), Box<dyn Error>> {
     let mut opts = getopts::Options::new();
     let opts = opts
-        .optflag("v", "verbose", "Always show the entire output.")
+        .optflag("", "git", "Grab the input data from the local Git history.")
```

Comment:
Same as above.

Reply:
Same as above, too.  I do not get what you intend to express.

Reply:
Hope this cleared up with my other comments. 

Also:
> I think "Parses the Git history" is quite colloquial and maybe not even correct. I have never heard git commits being called the git history. If you have something, that does call it that, I'd be interested. 😄

So, if you find an example there, it would also resolve this.

Reply:
See above.